docs/howto/write-logs.rst

```diff
@@ -110,6 +110,11 @@ To enable this feature, ``airflow.cfg`` must be configured as follows:
 
 In the above example, Airflow will try to use ``S3Hook('MyS3Conn')``.
 
+If you are using LocalStack to emulate Amazon S3, you must create this entry in airflow admin console:
+- Conn Id: MyS3Conn
+- Conn Type: S3
+- Extra: `{"host": "http://localstack:4572"}`
+
```

Comment:
```
You can also use `LocalStack <https://localstack.cloud/>`_ to emulate Amazon S3 locally.
To configure it, you only need to explicitly set the endpoint url. So that it points to your local stack. You can do this via the Connection Extra ``host`` field.
For example: ``{"host": "http://localstack:4572"}``
```

Reply:
This is the only setting that needs to be done, right?

Reply:
The remote logging configuration is the same?

Reply:
I am wondering if this should be an extra section or not. If it is really only the host setting that needs to be added to get this working I would suggest to keep it in the same section.
If not, we should explain the whole configuration from the beggining.